tests/slist_ut/slist_ut.c

```diff
@@ -0,0 +1,1114 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdbool.h>
+#include <stdlib.h>
+
+#include "macro_utils/macro_utils.h"
+#include "umock_c/umock_c.h"
```

Comment:
Let's have a newline between the headers of different libs we use

Reply:
done